docs/releases/2.2.txt

```diff
@@ -282,6 +282,9 @@ Database backend API
 Miscellaneous
 -------------
 
+* Admin's model history change messages will prefer usage of human-readable
```

Comment:
Should this come under a `django.contrib.admin` heading above? (Again, not sure on convention here.)

Reply:
I'm not sure about this either. @timgraham what are your thoughts on this?

Reply:
Thanks! I changed the message.

I believe this could be a potential breaking change if someone would be relying on iterating the dictionary of change messages of form for something.

One use-case I can think of is that being used in a test, but I believe `change_messages` can potentially have been used in code to trigger actions.

What do you think?

Reply:
I've added it currently in 3.0.txt breaking change section, but if you believe it's not a breaking change, then please let me know.

Thanks!